src/main/java/interface_adapter/graph/GraphPresenter.java

```diff
@@ -18,8 +20,10 @@ public GraphPresenter(GraphViewModel viewModel, ViewManagerModel viewManagerMode
 
     @Override
     public void prepareSuccessView(GraphOutputData data) {
-        // TODO implement this to change Journey view to the clicked Nodes page
-//        viewManagerModel.setState(journeyViewModel.getViewName());
-//        viewManagerModel.firePropertyChanged();
+        viewManagerModel.setState(journeyViewModel.getViewName());
```

Comment:
I'd maybe move the viewManagerModel down and have journey view switch first but it doesn't matter